### ISSUE TITLE
Crash when autouploading

### ISSUE THREAD

**Issue by arjenanker** [nextcloud/android#5995]
### Steps to reproduce
1. Set large folder to auto upload
2. See the app crash continuosly

### Expected behaviour
- The folder should fully upload

### Actual behaviour
- The app has been crashing for days and the amount of conflicts keeps increasing too, which makes the crashing worse

************ CAUSE OF ERROR ************

java.lang.IllegalStateException: Couldn't read row 1918, col 1 from CursorWindow.  Make sure the Cursor is initialized correctly before accessing data from it.
	at android.database.CursorWindow.nativeGetString(Native Method)
	at android.database.CursorWindow.getString(CursorWindow.java:469)
	at android.database.AbstractWindowedCursor.getString(AbstractWindowedCursor.java:53)
	at android.database.CursorWrapper.getString(CursorWrapper.java:141)
	at com.owncloud.android.datamodel.UploadsStorageManager.createOCUploadFromCursor(UploadsStorageManager.java:321)
	at com.owncloud.android.datamodel.UploadsStorageManager.getUploads(UploadsStorageManager.java:302)
	at com.owncloud.android.datamodel.UploadsStorageManager.getFailedUploads(UploadsStorageManager.java:368)
	at com.owncloud.android.files.services.FileUploader.retryFailedUploads(FileUploader.java:1005)
	at com.owncloud.android.utils.FilesSyncHelper.lambda$restartJobsIfNeeded$0(FilesSyncHelper.java:232)
	at com.owncloud.android.utils.-$$Lambda$FilesSyncHelper$fa2LBpPOxRVQOm4F_Glue5u4hdM.run(Unknown Source:10)
	at java.lang.Thread.run(Thread.java:919)

************ APP INFORMATION ************
ID: com.nextcloud.client
Version: 30110190
Build flavor: gplay

************ DEVICE INFORMATION ************
Brand: OnePlus
Device: OnePlus7Pro
Model: GM1913
Id: QKQ1.190716.003
Product: OnePlus7Pro_EEA

************ FIRMWARE ************
SDK: 29
Release: 10
Incremental: 2003131900

**Comment by tobiasKaminsky** [comment]
Will be fixed via #5985

**Comment by greve** [comment]
I assume this is a duplicate?

```
************ CAUSE OF ERROR ************

java.lang.IllegalStateException: Couldn't read row 1898, col 1 from CursorWindow.  Make sure the Cursor is initialized correctly before accessing data from it.
 at android.database.CursorWindow.nativeGetString(Native Method)
 at android.database.CursorWindow.getString(CursorWindow.java:469)
 at android.database.AbstractWindowedCursor.getString(AbstractWindowedCursor.java:53)
 at android.database.CursorWrapper.getString(CursorWrapper.java:141)
 at com.owncloud.android.datamodel.UploadsStorageManager.createOCUploadFromCursor(UploadsStorageManager.java:321)
 at com.owncloud.android.datamodel.UploadsStorageManager.getUploads(UploadsStorageManager.java:302)
 at com.owncloud.android.datamodel.UploadsStorageManager.getFailedUploads(UploadsStorageManager.java:368)
 at com.owncloud.android.files.services.FileUploader.retryFailedUploads(FileUploader.java:1005)
 at com.owncloud.android.utils.FilesSyncHelper.lambda$restartJobsIfNeeded$0(FilesSyncHelper.java:232)
 at com.owncloud.android.utils.-$$Lambda$FilesSyncHelper$fa2LBpPOxRVQOm4F_Glue5u4hdM.run(Unknown Source:10)
 at java.lang.Thread.run(Thread.java:929)

************ APP INFORMATION ************
ID: com.nextcloud.client
Version: 30110190
Build flavor: generic

************ DEVICE INFORMATION ************
Brand: HUAWEI
Device: HWLYA
Model: LYA-L29
Id: HUAWEILYA-L29
Product: LYA-L29

************ FIRMWARE ************
SDK: 29
Release: 10
Incremental: 10.0.0.196C432
```
If not, please let me know and I'll try to provide further information.

**Comment by AlvaroBrey** [comment]
Should be closed by #6037
